tests/MyGame/Example/ArrayStruct.py

```diff
@@ -20,53 +20,59 @@ def Init(self, buf, pos):
     # ArrayStruct
     def A(self): return self._tab.Get(flatbuffers.number_types.Float32Flags, self._tab.Pos + flatbuffers.number_types.UOffsetTFlags.py_type(0))
     # ArrayStruct
-    def B(self): return [self._tab.Get(flatbuffers.number_types.Int32Flags, self._tab.Pos + flatbuffers.number_types.UOffsetTFlags.py_type(4 + i * 4)) for i in range(15)]
+    def B(self, j = None):
+        if j==None:
+            return [self._tab.Get(flatbuffers.number_types.Int32Flags, self._tab.Pos + flatbuffers.number_types.UOffsetTFlags.py_type(4 + i * 4)) for i in range(15)]
```

Comment:
can the `15` be defined in terms of `BLength()`? Or does python not optimize those away?

Reply:
We can define that to be BLength, originally I thought it didn't make much difference due to the nature of auto-generated code. I'm not sure if it causes extra overhead in python, as it's just returning a single number I expect that python should optimise it or the overhead is extremely minimal.

Reply:
This has been changed now so it calls the correct length method